docs/reference/python/index.md

```diff
@@ -0,0 +1,53 @@
+
+The only drawback of installing python directly on your machine is that upgrading/downgrading is more tedious to
+do if you ever need to.
+
+### Installing anaconda
+
+[Anaconda](https://www.anaconda.com/) is a python and R distribution that aims to provide everything that you would
+need for data science. This includes:
+
+- The Python and R languages
+- Python and R libraries
+- Common IDEs like VSCode, PyCharm, Jupyter, Spyder, etc.
+- The `conda` package manager used for updating Anaconda packages
+
+This is not a standard tool used across Sailbot, but it is an option that is available to you. The main benefit of
+using Anaconda over installing python directly is that you can leverage virtual environments to change both your
+dependencies and your python version very quickly and effortlessly. The main drawback of using Anaconda is that it
+is clunky and can take up a fair amount of memory on your machine. Furthermore, you are also relying on a company
+for a major tool that you use (which can also be seen as a disadvantage depending on your preferences).
+
+## Python tutorials
+
+We understand that not everyone who joins Sailbot has Python in their toolkit, nor do we expect it either! Whether you
+are learning Python for the first time or you just want to brush up, we have provided some resources below. You **will not**
+learn absolutely everything from the resources below, but it is a good starting point. You will mostly learn through doing,
+as you would with most technical skills!
+
+| Resource               | Description                                                                                    |
+| :--------------------- | :--------------------------------------------------------------------------------------------- |
+| [The Python Tutorial](https://docs.python.org/3/tutorial/) | The official python tutorial. Good if you have some time on your hands and you are a completionist. Sections 1 - 5 and 9 are the most relevant. |
+| [w3schools Tutorial](https://www.w3schools.com/python/default.asp) | Good if you want a more brief introduction to Python. It breaks down a lot of concepts into sections. Everything up to Python Classes/Objects is relevant. |
+| [YouTube Tutorial](https://youtu.be/t8pPdKYpowI) | If you like video tutorials, then we recommend this tutorial. This video is about 5 hours long, but it pretty much covers everything that you'll need to know for Python and there are some hands on  projects. |
+| [Shorter YouTube Tutorial](https://youtu.be/kqtD5dpn9C8) | A shorter alternative YouTube tutorial condensed into 1 hour. It covers less material but still covers many of the essentials. |
+| [CodingBat Practice](https://codingbat.com/python) | Good resource to put your Python skills to practice on some simple coding problems. Note that this resource does not teach you python. |
+
+Feel free to use other resources other than the ones listed above if you find any that you like!
```

Comment:
You could change this sentence to something along the lines of "Feel free to add any resources that you found helpful to this list!"